package.json

```diff
@@ -28,9 +28,20 @@
     "commands": [
       {
         "command": "ehrql.debug",
-        "title": "OpenSAFELY: Debug ehrQL dataset"
+        "title": "OpenSAFELY: Debug ehrQL dataset",
+        "icon": "$(play)",
+        "enablement": "editorTextFocus && resourceLangId == python && isEhrqlFile"
```

Comment:
Do we want the `editorTextFocus` bit? It means if you have an ehrql file open, but your focus is on the debug output, or the terminal, then the debug ehrql disappears from the play button. It also means that "Debug ehrql" never appears in the command pallete because when that is open the text editor doesn't have focus. I think it's better without - unless there are edge cases I haven't thought of. If removed, should be removed from the menus bit below as well.

Reply:
Oh yes, I think you're right - we do want it to appear in the command pallete, I hadn't realised that the enablement config prevented that. And I can remove it from the menu config too, but it won't change the behaviour - the entire button disappears if the ehrql text editor doesn't have focus. That's what we want though, I think; it's the same behaviour you get from the python extension with a standard python file